pkg/kubernetes/version.go

```diff
@@ -4,4 +4,4 @@ package kubernetes
 const DefaultVersion = "1.16.11"
 
 // DefaultVersionsRange is the default Kubernetes versions' range used by WKS to validate Kubernetes versions.
-const DefaultVersionsRange = ">=1.14.1 <=1.17.5"
+const DefaultVersionsRange = ">=1.14.1 <=1.17.x"
```

Comment:
LGTM, but why is there `<=1.17.x` instead of plain `<=1.17` that I would have expected?
just because of the specific syntax we're using?

Reply:
I think so. From https://jubianchi.github.io/semver-check/#/1.17/1.17.4 they seem to be the same

> `1.17.x`
> X-Range constraint
> Constraint will be satisfied by versions matching >=1.17.0 <1.18.0.

> `1.17`
> Range constraint
> Constraint will be satisfied by versions matching >=1.17.0 <1.18.0.